scr/hw6/ArrayDataException.java

```diff
@@ -0,0 +1,13 @@
+package hw6;
+
+public class ArrayDataException extends Throwable implements MyException {
+    private String message;
+
+    public ArrayDataException(String m) {
+        message = m;
+    }
+
+    public void PrintMessage() {
```

Comment:
You still don't install sonarLint. It will help you. Just do it :).  Method names should be from lower case

Reply:
corrected printMessage